lib/buffer.js

```diff
@@ -852,6 +852,12 @@ function checkOffset(offset, ext, length) {
     throw new RangeError('Index out of range');
 }
 
+function checkByteLength(byteLength) {
+  if (typeof byteLength !== 'number' || byteLength <= 0)
+    throw new TypeError('"byteLength" must be a positive number ' +
```

Comment:
Thanks again for the feedback. I amended the proposed changes to the previous commit. I also updated the docs.